app/models/user.rb

```diff
@@ -6,7 +6,10 @@ class User
          :recoverable, :rememberable, :trackable, :validatable
 
   # Non-devise generated
-  field :name, type: String
+  field :first_name, type: String
+  field :last_name, type: String
+  field :line, type: String
+  field :role, type: String
```

Comment:
Devise and Mongo user models don't play perfectly nice together, it turns out,  and you have to specifically whitelist controller attributes. I'll handle in a separate commit.